dd-sdk-android/src/main/json/log/web-log-schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "web-log-schema.json",
+  "title": "WebLogEvent",
+  "type": "object",
+  "description": "Structure holding information about a Log",
```

Comment:
Because `browser` log events are not quite the same as ours and we have some extra required attributes. At deserialization it will create problems as it will try to find those attributes. Making them optional as I tried in the first place will modify the public API for `LogEvent` class and will create problems when using the Log event mapper.

Reply:
can we name that `webview-log-schema`? Web could be ambiguous.